_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and natural language processing (NLP) support tools for OpenSearch. It provides data analysts with the ability to:
+
+- Call OpenSearch indexes and manipulate using the opensearch-py-ml [DataFrame](https://opensearch-project.github.io/opensearch-py-ml/reference/dataframe.html) APIs. The opensearch-py-ml DataFrame wraps an OpenSearch index into a API similar to [pandas](https://pandas.pydata.org/), giving you the ability to process large amounts of data from OpenSearch inside a Jupyter Notebook.
```

Comment:
"and manipulate them"? I believe "notebook" should be lowercase when referring to the resource and not the product name.

Reply:
Using the standard made by the [Jupyter project](https://docs.jupyter.org/en/latest/running.html). I could switch to just "notebook", if you would prefer.

Reply:
It looks like even that page is inconsistent, but it does appear they capitalize it more than lowercase it, so let's keep as is.